### ISSUE TITLE
[4.0][a11y] Insufficient label contrast ratio at all switches

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#23863]
### Steps to reproduce the issue
1. Go to Backend, for example on the Global Configuration page or on the Modules > Site Modules > Article Category
2. Check the color contrast of the switch labels
![image](https://user-images.githubusercontent.com/29527200/52536835-c4e51000-2d5f-11e9-8700-916fb424d924.png)

### Expected result
Minimum required contrast ratio >= 4,5 
### Actual result
The labels of the options set by the switches have too low contrast to the background:
On the **Global Configuration** page the color combination #868e96/#ebeff5 has a contrast ratio of 2,88. Consider using the following foreground/background combination: #636e79/#ebeff5 / has a ratio of 4.51 
On the **Modules: Article Category** page the color combination #868e96/#fefefe has a contrast ratio of 3.29, which is not sufficient.  Consider using the following foreground/background combination: #6d7781/#fefefe  /  has a ratio of 4.52

**Comment by brianteeman** [comment]
Shouldnt this be posted in the private template repo?

**Comment by zwiastunsw** [comment]
Sorry, but.... where is the private template repository?

**Comment by brianteeman** [comment]
I have no idea. Obviously I was mistaken in assuming that when they said they had accessibility experts that it included the JAT

**Comment by puneeth-chanda** [comment]
can I work on this issue

**Comment by Quy** [comment]
@puneeth2001 You do not need to ask for permission. Go for it.

**Comment by puneeth-chanda** [comment]
@Quy  is it ok:
![screenshot from 2019-02-17 15-39-46](https://user-images.githubusercontent.com/43813460/52911477-4ee62900-32ca-11e9-9d9e-e3d8011a8353.png)

**Comment by zwiastunsw** [comment]
@puneeth2001 : Don't ask if it's okay. Do PR. Testers will evaluate.

**Comment by wilsonge** [comment]
This should have been fixed by https://github.com/joomla/joomla-cms/commit/609b77dcc2c930b797363c7550efb71ed08b391e
